io.github.philipk.boilr.yml

```
@@ -0,0 +1,70 @@
+app-id: io.github.philipk.boilr
+command: boilr
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    # To pick up vendored deps
+    CARGO_HOME: /run/build/boilr/cargo
+    RUSTFLAGS: "-L /app/lib/"
+
+command: boilr
+finish-args:
+  - --filesystem=xdg-data/Steam:rw
+  - --filesystem=~/.steam:rw # Steam (Non-flatpak)
+  - --filesystem=~/.var/app/com.valvesoftware.Steam:rw # Steam (Flatpak)
+  - --filesystem=xdg-data/lutris:rw # Lutris (non-flatpak)
+  - --filesystem=~/.var/app/net.lutris.Lutris:rw # Lutris (Flatpak)
+  - --filesystem=~/Games/Heroic:rw # Heroic (non-flatpak)
+  - --filesystem=~/.var/app/com.heroicgameslauncher.hgl:rw # Heroic (Flatpak)
+  - --filesystem=~/.config/itch:rw # Itch
+
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+
+  - --env=PATH=/app/bin:/usr/bin
+
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: boilr
+    buildsystem: simple
+    sources:
+      - type: dir
+        path: ../
+      - cargo-lock.json
+    build-commands:
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --verbose
+      - install -Dm755 ./target/release/boilr -t /app/bin/
+
+  - name: boilr-icon
+    buildsystem: simple
+    sources:
+      - type: file
+        path: ../resources/io.github.philipk.boilr.png
+    build-commands:
+      - install -Dm644 io.github.philipk.boilr.png  -t /app/share/icons/hicolor/32x32/apps
+
+  - name: boilr-desktop
+    buildsystem: simple
+    sources:
+      - type: file
+        path: io.github.philipk.boilr.desktop
+    build-commands:
+      - install -Dm644 io.github.philipk.boilr.desktop -t /app/share/applications/
+
+  - name: boilr-appinfo
+    buildsystem: simple
+    sources:
+      - type: file
+        path: io.github.philipk.boilr.appdata.xml
+    build-commands:
+      - install -Dm644 io.github.philipk.boilr.appdata.xml -t /app/share/metainfo/
```

```suggestion
      - cargo-lock.json
      - type: file
        path: io.github.philipk.boilr.desktop
      - type: file
        path: io.github.philipk.boilr.appdata.xml
    build-commands:
      - cargo --offline fetch --manifest-path Cargo.toml --verbose
      - cargo --offline build --release --verbose
      - install -Dm755 ./target/release/boilr -t /app/bin/
      - install -Dm644 resources/io.github.philipk.boilr.png  -t /app/share/icons/hicolor/32x32/apps
      - install -Dm644 io.github.philipk.boilr.desktop -t /app/share/applications/
      - install -Dm644 io.github.philipk.boilr.appdata.xml -t /app/share/metainfo/      
```


Also 32x32 is too small for icons.


And for a future release the .desktop and .appdata.xml should be managed upstream and installed from there.